.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -45,17 +45,21 @@ env:
   PR_LABELS: ${{ toJson(github.event.pull_request.labels.*.name) }}
 
 concurrency:
-  group: !{{ build_environment }}-${{ github.event.pull_request.number || github.sha }}
+  group: !{{ build_environment }}-${{ github.event.pull_request.number || github.sha }}-${{ github.event_name == 'workflow_dispatch' }}
   cancel-in-progress: true
 
```

Comment:
not necessary for this PR but it'd be good to move the concurrency group stuff into a macro